cips/cip-blob-verified-signer.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Support authored blobs
```

Comment:
[optional] this title is a bit unclear because blobs are already authored by blob submitters. An alternative title that may be a bit more self-evident is: "Include signer in share format"

Reply:
But blobs don't have the author / signer in the blob itself. It's embedded in the PFB. And this isn't just about changes to the share format

Reply:
Good point. An alternative name could be: "Blob v2 with signer"